src/Graph/Action.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+namespace Graph {	
+	struct ActionActor{
+	public:
+		float stimulation;
+		std::vector<std::string> requirements;
+		std::vector<int> strippingSlots;
+	};
+	struct ActionPerformer {
+	public:
+		std::unordered_map<std::string, float> floats;
+	};
```

Comment:
I don't think ActionPerformer needs to be an extra struct. Performers should be action actors just as actors and targets. So that all positions in an action can have a table of floats. Additionally a table of strings and a table of ints could be added for the ability to store more generic data if needed.